swaggerpay/payswaggerlatest.json

```diff
@@ -1121,6 +1200,89 @@
         },
         "additionalProperties": false
       },
+      "AMCSPayGetAllPaymentResponseDetails": {
+        "type": "object",
+        "properties": {
+          "customerId": {
+            "type": "integer",
+            "nullable": true,
+            "example": "1"
+          },
+          "customerName": {
+            "type": "string",
+            "nullable": true,
+            "example": "TestUser"
+          },
+          "ARAccountCode": {
+            "type": "string",
+            "nullable": true,
+            "example": "ACC1001"
+          },
+          "currency": {
+            "type": "string",
+            "nullable": true,
+            "example": "EUR"
+          },
+          "amount": {
+            "type": "Decimal",
+            "nullable": true,
+            "example": "10.02"
```

Comment:
I think we can remove the double quote ("") for this type

Reply:
done